packages/server/src/api/controllers/table/index.ts

```diff
@@ -77,11 +77,6 @@ export async function save(ctx: UserCtx<SaveTableRequest, SaveTableResponse>) {
   const renaming = ctx.request.body._rename
 
   const api = pickApi({ table })
-  // do not pass _rename or _add if saving to CouchDB
-  if (api === internal) {
```

Comment:
Might have missed this in one of the PRs - why are we not removing either of these props anymore?

Reply:
I removed it from here, moved the "clean" to `packages/server/src/api/controllers/table/internal.ts`, to keep it part of its responsability